qadence/backends/pyqtorch/convert_ops.py

```diff
@@ -291,23 +292,25 @@ def __init__(
                 use_full_support=False,
             )
             hmat = hmat.permute(1, 2, 0)
-            self._hamiltonian = lambda x: hmat
+            self.register_buffer("hmat", hmat)
+            self._hamiltonian = lambda self, values: self.hmat
```

Comment:
So I understand that the `self.register_buffer("hmat", hmat)` also implicitlly makes `self.hmat` defined?

Reply:
from the torch docs "Buffers can be accessed as attributes using given names."